refs/w3c.json

```
@@ -21662,6 +21713,9 @@
         "edDraft": "https://drafts.csswg.org/css-gcpm/",
         "repository": "https://github.com/w3c/csswg-drafts"
     },
+    "css-grid": {
+        "aliasOf": "css-grid-2"
```

```suggestion
        "aliasOf": "css-grid-1"
```

(grid-2 is still a delta spec for now)

---

Ok, I added a "overrides" map to the script to handle these kinds of cases.